macros/seed_existance_check.sql

```diff
@@ -0,0 +1,38 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{% macro seed_existance_check() %}
+    {% if execute %}
+        {# Ensure that this check is only performed during 'run' or 'build' commands #}
+        {%- if flags.WHICH in ('run') -%}
+            {% for node in graph.nodes.values() | selectattr("resource_type", "equalto", "seed") %}
+
+                {# Convert schema and table names to uppercase to prevent case sensitivity issues #}
+                {% set schema = node.schema | upper %}
+                {% set table = node.alias | upper %}
+                
+                {# Construct a SQL query to check if the seed table exists in the information schema #}
+                {% set check_table_exists_query = """SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = '" ~ schema ~ "' AND table_name = '" ~ table ~ "'" %}
+                
```

Comment:
Unfortunately databricks doesn't have an information schema in a hive metastore (welcome to the joys of supporting 4 warehouses!). In this case I would actually recommend using the dbt_utils.get_relations_by_pattern (https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#get_relations_by_pattern-source) macro to save re-inventing the wheel and process the results of this instead. This avoid _us_ having to manage mulit-warehouse support directly.